db/migrate/20210910082102_create_questions.rb

```diff
@@ -0,0 +1,12 @@
+class CreateQuestions < ActiveRecord::Migration[6.1]
+  def change
+    create_table :questions do |t|
+      t.string :heading
+      t.string :description
+      t.integer :votes
+      t.references :user, null: false, foreign_key: true
```

Comment:
references always in the starting

 null false passed explicitly?

Reply:
> references always in the starting

This was auto generated, I'll update the files

> null false passed explicitly?

No. autogenerated